sim_config.yaml

```diff
@@ -35,5 +35,12 @@
     detector_q_efficiency: 0.5         # detector quantum efficiency
     mtf_params: [0,0,1,0,0]            # to be described. [0,0,1,0,0] is perfect detector.
 
+  geometry_parameters:
+    n_samples: 4                       # number of samples to be generated
+
+  ctf_parameters:
+    distribution_type: 'uniform'        # type of distribution, either 'gaussian' or 'uniform'
+    distribution_parameters: [0, 1]     # distribution parameters, uniform: (low, high), gaussian: (loc, scale)
```

Comment:
maybe have
```
loc=0
scale=1
```

explicitly

Reply:
This change might be better in a future PR as it goes hand in hand with the comment above. I thought it'd be nice to brainstorm a structure that is a bit more verbose , so I added your comments to the issue here : https://github.com/compSPI/simSPI/issues/35